docs/docs/faq.md

```diff
@@ -25,6 +25,16 @@ On the other hand, you can still add whatever you want as a middleware which wra
 
 As of now, No. But it should not be a big issue because the reverse-proxy which gRPC-Gateway generates usually works as an independent process and communicates with your gRPC server over TCP or a Unix domain sockets (Unix systems only).
 
+If you do insist that your gateway should be written in the same language
+as your server, there are a few projects that will help with this:
+
+* NodeJS:
+  * [grps-dynamic-gateway](https://github.com/konsumer/grpc-dynamic-gateway)
+* TypeScript / Javascript
+  * Not a direct analog, but [rest2grpc](https://www.npmjs.com/package/rest2grpc) is similar.
+* Other languages
+  * Please add any references to other language implementations here.
+
```

Comment:
I think this is pretty hidden from users, I think it would be better with a new, separate page for "related projects". Could you add a new file under `docs/docs/related_projects.md` instead please?

Reply:
I've moved it.